### PR TITLE
Add No griefing rule

### PR DESCRIPTION
**Pull request by Gaming32**

### CODE REVIEW

File: yaml-files/welcome.yaml [hunk 1]
```diff
@@ -34,13 +34,9 @@
...
 
-            - name: 6. Don't abuse the rep system.
+            - name: 6. No griefing.
               value:
-                  "The rep system lets everyone know your 'reputation' on the server. If you've helped other people, they can upvote you, and
-                  if you've done something not so helpful like griefing, killing, etc. they can downvote you. Each action has to have a reason specified.
-                  Invalid reasons are things like 'being nice' and 'being mean', whereas valid reasons are 'they killed me and took my items' or
-                  'they helped me build my base'. Admins can check these logs, so be reasonable, or else we may have to take action against you,
-                  or reset their rep to the previous value. Fabric Reputation System can be found [here](https://modrinth.com/mod/fabric-rep-system)."
+                  "Griefing of any kind not enjoyed by the person getting griefed, including destroying of creations and stealing, is not allowed."
```
Comment (jadelily18) [inline yaml-files/welcome.yaml]: i think `not enjoyed by the person getting greifed` should be changed to `not explicitly permitted by the person getting griefed`

i also think `including destroying of creations and stealing` should be replaced with `including destruction of another person's builds or creations, as well as stealing`

**Review by jadelily18 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Why should stealing specifically be disallowed? People can hide their valuables if they don't trust those around - it's not as hard as protesting a whole build, and if a deep-buried underground stash gets stolen from it's basically covered by the no hacking rule.

Griefing needs to be disallowed as a rule because the time and effort required to destroy is never even comparably as much as to create. With stealing though, it's more complicated. There is no creative work in materials in a chest, and again such a chest can well be hidden in a place where it's humanly impossible to find without cheats - this makes _the time/effort of obtaining and stealing balanced_, and suddenly it's part of the fun. 

Also, if you don't want players to be able to conflict and do harm to each other in-game, you should disable PvP. Otherwise, frustration caused by stealing is part of the experience, it's what heats up the server and moves its history, and again, there is basically no history without conflict. **Players need to be allowed to engage in agressive ways for the server to be fun.**

**Comment by kuylar** [comment]
@HyperSoop 

> There is no creative work in materials in a chest

People still spend time obtaining the items in the chest, and most of the time, the materials in the chest are used in the creative work that will happen very soon in the future

> this makes the time/effort of obtaining and stealing balanced, and suddenly it's part of the fun.

For some reason, when *anyone's* items get stolen, they're not having fun.

Griefing & stealing is basically the same thing. Get over it.

**Review by kuylar — APPROVED** [review]

**Comment by ghost** [comment]
~~fish boi on their way to cancel soop~~

> People still spend time obtaining the items in the chest, and most of the time, the materials in the chest are used in the creative work that will happen very soon in the future

You can spend very little additional time to protect your items. Locks. They also exist in the modpack afaik. After all, just stash your stuff, hide your base, etc. And it's not that huge of a deal either, we don't have the kind of community that would massively steal from each other anyway - stealing never was an issue.

> For some reason, when anyone's items get stolen, they're not having fun.

It's part of it. Next time they'll know to hide their more valuable items, they and their friends'll try finding the thief for revenge, sparkling fun conflict for everyone. They can steal their stuff back even.

Being constantly watched by some eye in the sky with a banhammer is also not fun. If we don't want players doing something, we should add a mod or make another change to prevent it through gameplay, not make a written rule enforced only through immersion-breaking bans. If it's impossible to do so, then yes, it's time to make a rule. But in this case, you stand for fixing an issue that never even existed. When it comes to making such written rules - if it ain't broke, don't fix it.

**Comment by ghost** [comment]
(@kuylar)

**Comment by Gaming32** [comment]
There was a reason I mentioned enjoyment. If someone does enjoy getting stolen from, nothing will happen.

**Comment by kuylar** [comment]
> You can spend very little additional time to protect your items. Locks. 

Most players know that locks aren't reliable, since hoppers bypass them, and players can still break the chests.

> stealing never was an issue.

If stealing never was an issue, then why are you caring about it being banned?

> Next time they'll know to hide their more valuable items, they and their friends'll try finding the thief for revenge, sparkling fun conflict for everyone. 

For some unknown reason, the server has a mod that lets people become overpowered real easily *\*cough\* playerex \*cough\** (im aware that we're trying to cap the max level, but that's not the case here.). So not everyone can take revenge.

No matter how many points you put out, our view on stealing (& griefing) won't change. If you're not happy, you can just open your own server with this modpack and make your own rules. (hell, your server __might__ even get featured in the readme)

**Comment by jadelily18** [comment]
I see zero reason not to ban stealing. Unless players consent to it, all it does is ruin the fun for people having their stuff stolen.

File: yaml-files/welcome.yaml (hunk 1 above)
Comment (Gaming32) [reply yaml-files/welcome.yaml]: How's this?
```suggestion
                  "Griefing of any kind not explicitly permitted by the person getting griefed, including destruction of another person's builds or creations, as well as stealing, is not allowed."
```

**Comment by ghost** [comment]
> If stealing never was an issue, then why are you caring about it being banned?

> Being constantly watched by some eye in the sky with a banhammer is also not fun.  

Freedom is important for me, that's why. I want to be sure that if i do want to steal (or someone wants to steal from me) we can make our own story without admin intervention.

> There was a reason I mentioned enjoyment. If someone does enjoy getting stolen from, nothing will happen.

Nobody enjoys getting stolen from - but getting revenge can be enjoyable and engaging. If you have the option of using admins as your weapon of revenge, you'd be a fool not to use it. 

The idea of "Consent to stealing" makes this server a roleplay server, which it's not. Stealing is not something that would naturally happen with consent anyway.

File: yaml-files/welcome.yaml (hunk 1 above)
Comment (jadelily18) [reply yaml-files/welcome.yaml]: based

**Review by jadelily18 — APPROVED** [review]

**Comment by EmmaTheGiant** [comment]
Ive always been on the side of allowing griefing and stealing on the server. Without extra tools that allow admins to know who actually did the grief, the rule would be nothing more than an agreement among the people of the server.

I think there is an argument to be made that griefing destroys creativity, and ultimately ruins the fun of the server.  It stops people from building near spawn, and makes the server feel barren.

However, seeing as how we have the ability to hold thousands of stacks of items on us at any given moment, I see no reason to disallow stealing. If you are that worried about losing your items, use an ender pouch and backpacks, or hide them in a stash tens of thousands of blocks out. With waystones, backpacks, origins (shulker), ender chests, and a fairly large world, I see no reason stealing should even be a problem.

**Review by intergrav — APPROVED** [review]
t

**Comment by Gaming32** [comment]
For stealing, I definitely do see the points of not explicitly disallowing it. There is a reason I made this a PR, after all, instead of just committing directly to `main`.

**Comment by deniz-blue** [comment]
if we decide to ban griefing and stealing i want some sort of court to be integrated idk

"see you in court" or something idk would be fun

**Comment by deniz-blue** [comment]
> Being constantly watched by some eye in the sky with a banhammer is also not fun. If we don't want players doing something, we should add a mod or make another change to prevent it through gameplay, not make a written rule enforced only through immersion-breaking bans.

This implies that the punishment for stealing and griefing would be automatic.

if thats so, my suggestion would be that the victim of the griefing/stealing would have to report to the authority, which does kinda seem like the court joke above lol

**Comment by ghost** [comment]
~~Make the jail an in game location which is breakable out of~~

**Comment by Gaming32** [comment]
The trouble is finding who did the grief, since there are multiple ways for them to cover their tracks.

**Comment by ghost** [comment]
So, we have griefing - no, stealing - yesn't

then yes this is the next (non) issue; you can just install [Ledger](https://modrinth.com/mod/ledger) on the server that works

**Comment by Gaming32** [comment]
Ok, so, let's revive this. I think no griefing and no stealing is good. I think we should go with Ledger, at least till I finish my thing.

**Comment by ghost** [comment]
No griefing is definitely fine, but it should all be a moderator's discretion, not at "if you steal anything you get banned" - only major of repeaded cases should be punished

**Comment by Gaming32** [comment]
That is effectively how the rule is worded. If you steal and it's like really bad, then it should get punished. Like stealing someone's prized whatever, then yeeting it in lava or something.

**Comment by deniz-blue** [comment]
Yeah I think thats the sweet point, it allows for petty small crimes and doesnt excuse the really devestating losses, i approv 🐱👍

**Pull request merged**